polytope/polytope.py

```diff
@@ -495,43 +499,47 @@ def _translate(polyreg, d):
 
 
 def _rotate(polyreg, u=None, v=None, theta=None, R=None):
-    """Rotate this polyreg in place. Return the rotation matrix.
-
-    Simple rotations, by definition, occur only in 2 of the N dimensions; the
-    other N-2 dimensions are invariant with the rotation. Any rotations thus
-    require specification of the plane(s) of rotation. This function has three
-    different ways to specify this plane:
-
-    (1) Providing the indices [0, N) of the orthogonal basis vectors which
-    define the plane of rotation and an angle of rotation (theta) between them.
-    This allows easy construction of the Givens rotation matrix. The right hand
-    rule defines the positive rotation direction.
-
-    (2) Providing two unit vectors with no angle. The two vectors are contained
-    within a plane and the degree of rotation is the angle which moves the
-    first vector, u, into alignment with the second vector, v. NOTE: This
-    method is not implemented at this time.
-
-    (3) Providing an NxN rotation matrix, R. WARNING: No checks are made to
+    """Rotate C{polyreg} in-place. Return the rotation matrix.
+
+    There are two types of rotation: simple and compound. For simple rotations,
+    by definition, all motion can be projected as circles in a single plane;
+    the other N-2 dimensions are invariant. Therefore any simple rotation can
+    be parameterized by its plane of rotation. Compound rotations are the
+    combination of multiple simple rotations; they have more than one plane of
+    rotation. For N > 3 dimensions, a compound rotation may be necessary to map
+    one orientation to another (Euler's rotation theory no longer applies).
```

Comment:
would "Euler's rotation theorem" be more appropriate here, rather than "Euler's rotation theory"?